adobe/spectrum-css
# feat(colorslider): S2 migration

## Description

`CSS-1028`

# S2 colorslider migration

This migrates the `colorslider` component to S2. Custom properties have been remapped per the design spec.

| Before                    | After                      |
| ------------------------- | -------------------------- |
| `--spectrum-gray-900-rgb` | `--spectrum-gray-1000-rgb` |


### Validation steps

1. Open the URL for the PR.
2. Navigate to the `colorslider` component and verify no regressions have occurred.

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:


2. If components have been modified, VRTs have been run on this branch:


## Screenshots



## To-do list
